src/main/scala/com/wavesplatform/Application.scala

```diff
@@ -249,46 +246,63 @@ object Application extends ScorexLogging {
-
-          Metrics.write(
-            Point
-              .measurement("config")
-              .addField("miner-micro-block-interval", miner.microBlockInterval.toMillis)
-              .addField("miner-max-transactions-in-key-block", miner.maxTransactionsInKeyBlock)
-              .addField("miner-max-transactions-in-micro-block", miner.maxTransactionsInMicroBlock)
-              .addField("miner-min-micro-block-age", miner.minMicroBlockAge.toMillis)
-              .addField("mbs-wait-response-timeout", microBlockSynchronizer.waitResponseTimeout.toMillis)
-          )
+    log.trace(s"System property sun.net.inetaddr.ttl=${
+      System.getProperty("sun.net.inetaddr.ttl")
+    }")
+    log.trace(s"System property sun.net.inetaddr.negative.ttl=${
+      System.getProperty("sun.net.inetaddr.negative.ttl")
+    }")
+    log.trace(s"Security property networkaddress.cache.ttl=${
+      Security.getProperty("networkaddress.cache.ttl")
+    }")
+    log.trace(s"Security property networkaddress.cache.negative.ttl=${
+      Security.getProperty("networkaddress.cache.negative.ttl")
+    }")
+
+    RootActorSystem.start("wavesplatform", config) {
+      actorSystem =>
+
+        import actorSystem.dispatcher
+
+        isMetricsStarted.foreach {
+          started =>
+            if (started) {
+
+              import settings.synchronizationSettings.microBlockSynchronizer
+              import settings.{minerSettings => miner}
+
+              Metrics.write(
+                Point
+                  .measurement("config")
+                  .addField("miner-micro-block-interval", miner.microBlockInterval.toMillis)
+                  .addField("miner-max-transactions-in-key-block", miner.maxTransactionsInKeyBlock)
+                  .addField("miner-max-transactions-in-micro-block", miner.maxTransactionsInMicroBlock)
+                  .addField("miner-min-micro-block-age", miner.minMicroBlockAge.toMillis)
+                  .addField("mbs-wait-response-timeout", microBlockSynchronizer.waitResponseTimeout.toMillis)
+              )
+            }
         }
-      }
-
-      configureLogging(settings)
 
-      // Initialize global var with actual address scheme
-      AddressScheme.current = new AddressScheme {
-        override val chainId: Byte = settings.blockchainSettings.addressSchemeCharacter.toByte
-      }
+        configureLogging(settings)
 
-      log.info(s"${Constants.AgentName} Blockchain Id: ${settings.blockchainSettings.addressSchemeCharacter}")
-
-      new Application(actorSystem, settings, config.root()) {
-        override def shutdown(): Unit = {
-          Kamon.shutdown()
-          Metrics.shutdown()
-          super.shutdown()
+        // Initialize global var with actual address scheme
+        AddressScheme.current = new AddressScheme {
+          override val chainId: Byte = settings.blockchainSettings.addressSchemeCharacter.toByte
         }
-      }.run()
+
+        log.info(s"${
```

Comment:
This weird formatting happens when you format incorrect with IDEA while syntax is incorrect in terms of `{` and `}`